### ISSUE TITLE
On-screen keyboard for Windows version

### ISSUE THREAD

**Issue by morosevacuum** [Stremio/stremio-features#545]
**Describe the solution you'd like**
Another request from this HTPC user. I'm not certain if the Android TV version of Stremio has this feature, though I have to assume it does. In any case, I would love to be able to navigate to the search bar using arrow keys and for there to be an on-screen keyboard upon clicking into the search field. Maybe this could be an optional feature in the settings?

**Describe alternatives you've considered**
Have tried to use Steam's on screen keyboard, as well as Windows accessbility keyboard, but neither work quite as fluidly as I'd like them. Steam's doesn't seem to maintain focus over the Stremio window, and the Windows keyboard requires a great deal of customization in order to allow for its keyboard shortcut to be trigger by the press of a single button. In both cases, additional input is required to even bring the keyboard up on screen. Would love if there was an automated on-screen keyboard that appeared upon selecting the search field.

**Comment by TRtomasz** [comment]
No, we will not add on screen keyboard, but usually you can have an app for on screen keyboard
